README.md

```diff
@@ -24,6 +24,11 @@ macAddressList | String | List of MAC address should be set per host In the foll
 - Create the ign file from the custom-config.fcc: 
 ```
 fcct custom-config.fcc > file.ign
+```
+    **TIP:** To change the RHCOS sysroot partition size (currently set to 600G) you can use the following before creating the ign file:
+
+```
+sed -i -e "s/OFFSET=600G/OFFSET=120G/" custom-config.fcc
```

Comment:
@RamLavi  should we add this to the make file?

Reply:
@kobi86 good question

@jeniferh do we just want to notify on the README that we can change it or do we want to really change it?
if the answer is the latter then I wonder why aren't we just changing this on the custom-config.fcc?

Reply:
Ideally giving RHCOS 600GB is good practice since by default all container image storage (and Monitoring etc. if not configured otherwise) will use this space, I just wanted to note how that value could be changed in very space-constrained environments if needed.

Reply:
I see, so to answer @kobi86 I think that on `simplify-networking` branch this should be added to the README as a note and not to the makefile..